facia-tool/public/js/modules/content-api.js

```diff
@@ -186,13 +195,18 @@ function decorateBatch (articles) {
         }
 
         results.forEach(function(result) {
-            var icc = internalContentCode(result);
+            var pageCode = internalPageCode(result),
+                icc = internalContentCode(result);
 
-            if(icc) {
-                cache.put('contentApi', icc, result);
+            if (pageCode) {
+                cache.put('contentApi', pageCode, result);
+                if (icc) {
```

Comment:
could you enforce that with some kind of switch or test?  otherwise it might get left in the code for ever